### ISSUE TITLE
Als gemeente wil ik de flow van productdefinitie tot levering weten

### ISSUE THREAD

**Issue by frankmijn** [VNG-Realisatie/producttypecatalogus#41]
zodat ik het design pattern van een product in de pdc tot levering en hoe doe ik dat technisch gezien in elkaar past. 

**Definition of done**
- [ ] besluit relevant/generiek
- [ ] indien relevant/generiek; besluit verwerken in kernmodel of extensie
- [ ] indien kernmodel; verwerkt in model

**Comment by Martindebijl** [comment]
Frank, kun je hier een toelichting op geven? Ik interpreteer deze user story als het opnemen van een veld, waarin de relatie met/ verwijzing naar een database van design patterns wordt gelegd.

**Comment by frankmijn** [comment]
Ja, dat gaat over de relaties tussen de entiteiten. En waar het ook over gaat, is hoe de flow is over deze entiteiten heen. Dus geplot op aanvraag, levering etc.
